src/redux/utilities.js

```diff
@@ -0,0 +1,33 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const appID = 'dXhbHW84R2UylfqfKuq7';
+
+export const addBook = createAsyncThunk('Books/addBook', async (book) => {
+  try {
+    await axios.post(`https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/${appID}/books`, book);
+  } catch (e) {
+    throw new Error(e);
+  }
+  return book;
+});
```

Comment:
- [ ] When I try to add a book I get this noisy error from the app. And the book is not added to the UI, or API I think. Kindly fix that.

  ![image](https://github.com/katarighe/bookstore/assets/54848363/34b57a9a-b40a-4c3b-9841-9857e12f5fb0)